modules/web/src/app/wizard/step/cluster/component.ts

```diff
@@ -701,16 +702,19 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
     }
   }
 
-  private _setDefaultCNIVersion(): void {
+  private _setDefaultCNIVersion(cniDefaultVersion: string): void {
     if (this.cniPluginVersions.length > 0 && !this.form.get(Controls.CNIPluginVersion).value) {
       // Dual-stack not allowed on Canal CNI version lower than 3.22.
       if (this.controlValue(Controls.CNIPlugin) === CNIPlugin.Canal && this.isDualStackIPFamilySelected()) {
         this.cniPluginVersions = this.cniPluginVersions.filter(v =>
           gte(coerce(v), this._canalDualStackMinimumSupportedVersion)
         );
       }
-
-      this.form.get(Controls.CNIPluginVersion).setValue(this.cniPluginVersions[this.cniPluginVersions.length - 1]);
+      if (cniDefaultVersion) {
```

Comment:
We're filtering `cniPluginVersions` above so here we should also check if the default version exists in filtered list. For example, if `Canal` plugin is selected and the default version is not supported.